src/bootstrap/cc_detect.rs

```diff
@@ -47,6 +47,8 @@ fn cc2ar(cc: &Path, target: TargetSelection) -> Option<PathBuf> {
         Some(PathBuf::from("ar"))
     } else if target.contains("vxworks") {
         Some(PathBuf::from("wr-ar"))
+    } else if target.contains("android") {
+        Some(cc.parent().unwrap().join(PathBuf::from("llvm-ar")))
```

Comment:
Is this a platform limitation, android only ever supports LLVM and not GCC? or is this just what happens to work with your setup? If the latter, I think you should use the `target.ar` config setting instead.

Reply:
Correct, Android only supports LLVM and not GCC.  I'll ask if there is a reason we don't have a symlink or shell script in place.

Reply:
That's ok - if anyone building android would also need to use llvm-ar we can hardcode it here.